ext_localconf.php

```diff
@@ -37,6 +37,19 @@
   );
 }
 
+// New content element wizard icon
```

Comment:
I deliberately copied the icon of "content-elements-login" into hairu. If you install both extension, it might be confusing, if you see two similar icons. So I prepare all that you just can exchange the "login.svg" to personalize the hairu content element icon.